mozilla-release/mobile/android/base/java/org/mozilla/gecko/preferences/GeckoPreferences.java

```diff
@@ -844,9 +846,7 @@ public boolean onPreferenceClick(Preference preference) {
                 /* Cliqz start */
                 // Format the Human web link
                 else if (PREFS_HUMAN_WEB_LINK.equals(key)) {
-                    final String LOCALE = Locales.getLanguage(Locale.getDefault());
-
-                    final String url = getResources().getString(R.string.pref_human_web_url, LOCALE);
+                    final String url = getResources().getString(R.string.pref_human_web_url);
```

Comment:
why is the locale removed?

Reply:
because I removed the parameter from the Url.
when we have the translation of the urls it should be ok.
Our Urls have a special case in German Urls like.
https://cliqz.com/report-url
but English one or others
https://cliqz.com/en/report-url
also we don't have all locals.

so I think it's better to be solved by translated strings rather than replacing the parameter.

Reply:
I spoke with Christian, they have already a solution in place for this agreed before with the Desktop guys: you can use just https://cliqz.com/report-url, the website will detect the user language. Perhaps we should move this urls to the branding, it is highly possible we will use different urls for Ghostery